react/components/ProductMessagesExport.tsx

```diff
@@ -109,4 +155,4 @@ const ProductMessagesExport: FC<Props> = ({
   )
 }
 
-export default injectIntl(ProductMessagesExport)
+export default withToast(injectIntl(ProductMessagesExport))
```

Comment:
(No need for changes atm, just a reminder) I think this `withToast` is turning this component into an `any`, so Typescript can't typecheck it.

Reply:
Yes!
Left like this because didn't know how to type it, will try to do something based on `injectIntl` type:

```
function injectIntl<P>(component: React.ComponentType<P & InjectedIntlProps>, options?: InjectIntlConfig):
    React.ComponentClass<Pick<P, Exclude<keyof P, keyof InjectedIntlProps>>> & { WrappedComponent: React.ComponentType<P & InjectedIntlProps> };
```